public/game-master/styles/game-master.styl

```diff
@@ -21,8 +21,15 @@
 		
 		.round-title
 			flex-grow 4
-		.round-timer
-			flex-grow 1
+			line-height 1.2em
```

Comment:
Treat it as a recommendation from timer. it guarantees that it'll look nice with such font-size\line-height. But it's always possible to override. But yes, could be easily removed